packages/e2e/puppeteer/tests/location-component.ts

```diff
@@ -13,8 +21,8 @@ const testFlow = async (page: puppeteer.Page, options: FlowOptions) => {
   await page.click('[data-testid="next-button"]');
 
   await page.waitForSelector('[name="postalCode"]');
-  await page.type('[name="postalCode"]', address.zipCode);
-  await page.type('[name="houseNumber"]', address.houseNumberFull);
+  await page.type('[name="postalCode"]', address[0]);
+  await page.type('[name="houseNumberFull"]', address[1].toString());
```

Comment:
I think we need to improve on the positional values in the future. No more `address[0]` but `address.zipCode` etc.

Reply:
Agree